src/WebJobs.Extensions.DurableTask/ContextImplementations/RemoteOrchestratorContext.cs

```diff
@@ -182,5 +185,20 @@ private void SetResultInternal(OrchestratorExecutionResult result)
 
             this.executionResult = result;
         }
+
+        private void ValidateCustomStatusSize(string customStatus)
```

Comment:
Non-blocking suggestion: make this static. Some code analyzers will issue a warning for this.